src/gleam/dynamic.gleam

```diff
@@ -27,33 +27,45 @@ pub external fn from(a) -> Dynamic =
 pub external fn unsafe_coerce(Dynamic) -> a =
   "gleam_stdlib" "identity"
 
-/// Check to see whether a Dynamic value is a bit_string, and return the bit_string if
+external fn erl_string(from: Dynamic) -> Result(BitString, String) =
+  "gleam_stdlib" "decode_string"
+
+/// Check to see whether a Dynamic value is a string, and return the string if
 /// it is.
 ///
 /// ## Examples
 ///
-///    > bit_string(from("Hello")) == bit_string.from_string("Hello")
-///    True
+///    > string(from("Hello"))
+///    Ok("Hello")
 ///
-///    > bit_string(from(123))
-///    Error("Expected a BitString, got `123`")
+///    > string(from(123))
+///    Error("Expected a String, got `123`")
 ///
-pub external fn bit_string(from: Dynamic) -> Result(BitString, String) =
-  "gleam_stdlib" "decode_bit_string"
+pub fn string(from: Dynamic) -> Result(String, String) {
+  erl_string(from)
+  |> result.then(
+    fn(raw) {
+      case bit_string.is_utf8(raw) {
```

Comment:
I put this section before the definition of dynamic.bit_string otherwise I would need to give a different name to the bitstring module when it's imported.